### PR TITLE
Add link to the page

### PR DESCRIPTION
**Pull request by schmijos**

### CODE REVIEW

File: js/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Frontend apps
```
Comment (coorasse) [inline js/README.md]: It seems strange that the link states "Simple JS App" and the title of this page is a more generic "Frontend Apps"

File: js/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Trivial app
```
Comment (coorasse) [inline js/README.md]: I consider this section obvious, but probably it doesn't hurt to have it.

File: js/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Consider using only static files (example: [Green Card Predictor](https://github.com/renuo/green-card-probability-frontend))
+and CDNs.
+
+## Simple JS app
+
+If you need a really simple JavaScript application and you need more than just an *index.html*, you can checkout this template:
+https://github.com/renuo/tamedia-altersheime/tree/0a5fc447bc7fdae9b81bbe0918ca64c864c58354
```
Comment (coorasse) [inline js/README.md]: 👍  I cannot go too much in detail in the template. I'd like if also @macav could take a look at it. 
I am just wondering if, for such a simple app, cucumber is not an overkill.. 🤔

File: js/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+The rest is done in these three files:
+
+* index.html
+* index.css
+* index.js
+
+## Complex JS app
+
+Use eighter React or [Angular](../angular/README.md).
```
Comment (coorasse) [inline js/README.md]: `either`

File: js/README.md (hunk 3 above)
Comment (schmijos) [reply js/README.md]: I think cucumber is the easiest way to have tests at all. I think unit tests would be overkill.

File: js/README.md (hunk 1 above)
Comment (schmijos) [reply js/README.md]: Yes.

**Pull request merged**